tasks/state-installed.yml

```diff
@@ -41,6 +41,14 @@
 
 - import_tasks: build/install-k3s.yml
 
+- name: Ensure containerd installation tasks are run
+  block:
+    - include_tasks: build/containerd/registries.yml
+  when:
+    - k3s_registries is defined
```

Comment:
By default this file would be created but the contents would simply be:

```yaml
---

mirrors:
configs:
```

Presumably this is OK in your experience?

Reply:
yeah, it works